src/Computers.java

```diff
@@ -0,0 +1,61 @@
+public class CalculatingDevice {
+    protected String name;
+
+    CalculatingDevice(String name) {
+        this.name = name;
+    }
+    
+    public String calculate(double x, double y) {
```

Comment:
Please add a comment above explaining what this method do

Reply:
That's because you have multiple caclulate methods in the subclasses. It will be much faster to look at the comment to see what this method does